src/js/components/PositionItem/VoterPositionEditModal.jsx

```diff
@@ -0,0 +1,371 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Tooltip } from '@mui/material';
+import { withStyles } from '@mui/styles';
+import CreateOutlinedIcon from '@mui/icons-material/CreateOutlined';
+import DesignTokenColors from '../../common/components/Style/DesignTokenColors';
+import ModalDisplayTemplateA from '../Widgets/ModalDisplayTemplateA';
+
+const VoterPositionEditModal = ({ showVoterEdit, setShowVoterEdit, candidateName, voter, classes }) => {
+  const [opinionVisibility, setOpinionVisibility] = useState('Everyone');
+  const [endorseOption, setEndorseOption] = useState('Neutral');
+  const [opinionText, setOpinionText] = useState('');
+  const [isFormValid, setIsFormValid] = useState(false);
+  const [disableToolTip, setDisableToolTip] = useState(false);
+  const {voter_photo_url_medium, first_name, last_name, full_name} = voter;
```

Comment:
Hi @jedwardmook, I will approve this now, but normally we would convert these to camelCase like this:
const {
    voter_photo_url_medium: voterPhotoUrlMedium,
    first_name: firstName,
    last_name: lastName,
    full_name: fullName,
} = voter;